test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -422,23 +421,23 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		})
 	})
 
-	Context("Updating of nodeSelector parameter and node labels", func() {
+	Context("Updating of nodeSelector parameter and node labels", Ordered, func() {
 		var mcp *machineconfigv1.MachineConfigPool
 		var newCnfNode *corev1.Node
+		var oldMcpSelector, oldNodeSelector map[string]string
+		var removeLabels func(map[string]string, *corev1.Node) error
 
 		newRole := "worker-test"
 		newLabel := fmt.Sprintf("%s/%s", testutils.LabelRole, newRole)
 		var labelsDeletion = false
 		newNodeSelector := map[string]string{newLabel: ""}
 
-		//fetch the latest profile
-		profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
-		Expect(err).ToNot(HaveOccurred())
-		var oldMcpSelector, oldNodeSelector map[string]string
-		var removeLabels func(map[string]string, *corev1.Node) error
-
 		//fetch existing MCP Selector if exists in profile
 		BeforeEach(func() {
+			//fetch the latest profile
+			profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
```

Comment:
Is this addition related to the PR ?

Reply:
ginkgo v2 is failing if there is a expect in the context outisde of beforeall or it, move it here in order to fix it.